src/Core/Managed/Shared/Extensibility/Metrics/MetricTerms.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+
+namespace Microsoft.ApplicationInsights.Extensibility.Metrics
+{
+    internal static class MetricTerms
+    {
+        private const string MetricPropertiesNamePrefix = "Microsoft.ApplicationInsights.Metrics";
+        public static class Extraction
+        {
+            //Microsoft.ApplicationInsights.Metrics.MetricIsAutocollected
+            public static class ConsideredByProcessors
+            {
+                public static class Moniker
+                {
+                    public const string Key = MetricPropertiesNamePrefix + ".Extraction.ConsideredByProcessors";
+                }
+            }
+        }
+
+        public static class Autocollection
+        {
+            public static class Moniker
+            {
+                public const string Key = MetricPropertiesNamePrefix + ".MetricIsAutocollected";
+                public const string Value = "True";
+
+            }
+
+            public static class MetricNames
+            {
+                private const string PreviewLabel = " (Preview)";
+
+                public static class Requests
```

Comment:
nitpick: C# has namespaces, and long variable names, these nested classes are not a style I care for just to get dots.